ababot/bot/src/types/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod stonk;
```

Comment:
I imagine it's better to have a folder for every command, such that we can keep the structs for a given command with the functionality for the command

Reply:
Not in the scope of this PR